loads/loadconf.py

```diff
@@ -37,41 +31,37 @@ def read_file(owner, repo, filename='loads.json'):
 
 
 def loads_broker_install():
-    Log.header('INSTALL LOADS-BROKER')
-    """
-    process(
-        'rm -rf _temp/loads-broker',
-        'Cleaning up old loads-broker install'
-    )
-    """
-    process(
-        'git clone https://github.com/loads/loads-broker _temp/loads-broker',
-        'Installing loads-broker'
-    )
-    process(
-        'cd _temp/loads-broker;pip install -r test-requirements.txt',
-        'Installing loads-broker libraries'
-    )
-    process(
-        'cd _temp/loads-broker;python setup.py develop',
-        'Setting up loads-broker'
-    )
+    if not (os.path.isdir(PATH_LOADS_BROKER)):
+        Log.header('INSTALL LOADS-BROKER')
+        process(
+            'git clone https://github.com/loads/loads-broker \
+                 _temp/loads-broker',
+            'Installing loads-broker'
+        )
+        process(
+            'cd _temp/loads-broker;pip install -r test-requirements.txt',
+            'Installing loads-broker libraries'
+        )
+        process(
+            'cd _temp/loads-broker;python setup.py develop',
+            'Setting up loads-broker'
+        )
+
+
+def print_menu(plans):
+    Log.header('RUN MENU')
```

Comment:
RUN MENU header is running twice, once before run menu and once when we begin printing out logging from the loads-broker proc.